File: 03_스택_큐_재귀/01_stack.py
stack = []

stack.append(5)
stack.append(2)
stack.append(3)
stack.append(7)

stack.pop()

stack.append(1)
stack.append(4)

stack.pop()

print(stack)       # bottom 부터 원소 출력
print(stack[::-1]) # top부터 원소 출력

File: 03_스택_큐_재귀/01_stack.cpp
#include <iostream>
#include <stack>

using namespace std;

int main(void){

    stack <int> s;

    s.push(5);
    s.push(2);
    s.push(3);
    s.push(7);

    s.pop();

    s.push(1);
    s.push(4);

    s.pop();

    while (!s.empty())
    {
        cout << s.top() << ' '; // top에 있는 원소값 출력
        s.pop(); // 삭제
    }
    

    return 0;
}